src/controller.cpp

```diff
@@ -244,7 +298,7 @@ float Controller::PID::run(float dt, float x, float x_c, bool update_integrator)
   if (dt > 0.0001f) {
     // calculate D term (use dirty derivative if we don't have access to a measurement of the
     // derivative) The dirty derivative is a sort of low-pass filtered version of the derivative.
-    //// (Include reference to Dr. Beard's notes here)
+    //// TODO: (Include reference to Dr. Beard's notes here)
```

Comment:
Should we add these references before merging?

Reply:
Those were notes from before... I don't know what they were referencing, so maybe we remove that line?